cypress/support/step_definitions/annual-review/scheduler/the_string_key_date_email_will_be_sent_to_post.js

```diff
@@ -0,0 +1,45 @@
+Then("the {string} key date email is sent to post", function (keyDate) {
+  cy.task("db", {
+    operation: "list.findFirst",
+    variables: {
+      where: {
+        reference: "SMOKE",
+      },
+    },
+  }).then((list) => {
+    const keyDateReminders = {
+      START: "sendStartedPostEmail",
+      POST_ONE_DAY: "sendOneDayPostEmail",
+      POST_ONE_WEEK: "sendOneWeekPostEmail",
+      POST_ONE_MONTH: "sendOneMonthPostEmail",
```

Comment:
I think if you have time it would be good to run me through what these keys are doing. I've seen them before but haven't yet gone through the part of the code they affect.